auth-service/src/main/java/com/vlad/kuzhyr/authservice/utility/logger/LogUtils.java

```diff
@@ -0,0 +1,50 @@
+package com.vlad.kuzhyr.authservice.utility.logger;
+
+public class LogUtils {
+
+    public static String maskEmail(String email) {
```

Comment:
```suggestion
    public static String maskEmail(String email) {
        if (email == null || !email.contains("@")) return email;

        int atIndex = email.indexOf('@');
        String localPart = email.substring(0, Math.min(atIndex, 3));
        return localPart + "***" + email.substring(atIndex);
    }
```
остальные тоже можно оптимизировать